packages/cli/project.json

```diff
@@ -20,18 +20,11 @@
         ]
       }
     },
-    "sandbox": {
-      "executor": "@nx/jest:jest",
-      "outputs": ["{workspaceRoot}/coverage/packages/cli"],
-      "options": {
-        "jestConfig": "packages/cli/jest.config.ts"
-      }
-    },
```

Comment:
This is not the correct way to run E2E test! 

The sandbox should fully be deleted!

Additionally this would be a target for the e2e not for the lib itself.